### PR TITLE
Alphabetical changes

### PR DESCRIPTION
**Pull request by stubelshe**
Nothing major, just some rearranging.

### CODE REVIEW

File: src/prop-editor/prop-editor.scss [hunk 1]
```diff
@@ -28,6 +25,14 @@
     }
   }
 
+  .reset {
```
Comment (mvolkmann) [inline src/prop-editor/prop-editor.scss]: Move this to the bottom?

File: src/share.scss [hunk 2]
```diff
@@ -7,4 +7,4 @@ $gray1: #f4f4f4;
 $gray2: #ddd;
 $gray3: #ccc;
 $gray4: #5b6770;
-$gray5: #333;
+$gray5: #333;
```
Comment (mvolkmann) [inline src/share.scss:10]: If you run "npm run format", it will format all the .js and .scss files.
It always adds a blank line to all the files.

Comment (stubelshe) [reply src/share.scss:10]: Gotcha

File: src/prop-editor/prop-editor.scss (hunk 1 above)
Comment (stubelshe) [reply src/prop-editor/prop-editor.scss]: Moved.

**Pull request merged**